gatorgrade/output/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
+name = "poetry-demo"
+version = "0.1.0"
+description = ""
+authors = ["Me lol"]
```

Comment:
Can you put the authors in here?

Reply:
This poetry installation was mistakenly included, we were using it to get work done before poetry was installed in the main branch. The whole installation has been removed now!